src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,95 @@
-import { useState } from 'react';
+import React, { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
+type Props = {
+  onAdd: (movie: Movie) => void
+};
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
   // Increase the count after successful form submission
   // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+
+  const [count, setCount] = useState(0);
+
+  const [movieDef, setMovieDef] = useState({
+    title: '',
+    description: '',
+    imgUrl: '',
+    imdbUrl: '',
+    imdbId: '',
+  });
```

Comment:
move this object to a variable (outside NewMovie component)
```
const initialMovieState = {
  title: '',
  description: '',
  imgUrl: '',
  imdbUrl: '',
  imdbId: '',
}
```

```suggestion
  const [movieDef, setMovieDef] = useState(initialMovieState);
```